From 3573a04198b562a52de8870a8a7aff9af1c3c584 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 24 Jul 2018 15:35:20 +0100
Subject: [PATCH 3/8] vddk: Add comment about my experiment with
 PrepareForAccess.

(cherry picked from commit ba593d2dfa3b3ccd4073f7bad7bcd2d67ce23b64)
---
 plugins/vddk/vddk.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/plugins/vddk/vddk.c b/plugins/vddk/vddk.c
index e3190cd..77e3406 100644
--- a/plugins/vddk/vddk.c
+++ b/plugins/vddk/vddk.c
@@ -370,8 +370,9 @@ vddk_open (int readonly)
   }
 
   /* XXX Some documentation suggests we should call
-   * VixDiskLib_PrepareForAccess here.  However we need the true VM
-   * name to do that.
+   * VixDiskLib_PrepareForAccess here.  It may be required for
+   * Advanced Transport modes, but I could not make it work with
+   * either ESXi or vCenter servers.
    */
 
   DEBUG_CALL ("VixDiskLib_ConnectEx",
-- 
2.18.0

